report/src/main/webapp/package.json

```diff
@@ -1,29 +1,46 @@
 {
-  "name": "AET",
-  "description": "AET",
-  "devDependencies": {
-    "cheerio": "^0.19.0",
-    "connect-livereload": "^0.5.3",
-    "findup-sync": "~0.1.2",
-    "grunt": "^0.4.4",
-    "grunt-bower-task": "^0.4.0",
-    "grunt-cli": "^1.2.0",
-    "grunt-contrib-clean": "^1.0.0",
-    "grunt-contrib-compass": "^1.0.1",
-    "grunt-contrib-connect": "^0.9.0",
-    "grunt-contrib-copy": "^1.0.0",
-    "grunt-contrib-csslint": "~0.1.2",
-    "grunt-contrib-htmlmin": "~0.1.3",
-    "grunt-contrib-jshint": "~1.0.0",
-    "grunt-contrib-uglify": "~0.2.4",
-    "grunt-contrib-watch": "^0.6.1",
-    "grunt-includes": "^0.4.4",
-    "grunt-open": "^0.2.3",
-    "lodash": "^4.14.2",
-    "matchdep": "^0.3.0",
-    "time-grunt": "^1.0.0"
+  "name": "aet",
+  "version": "2.1.16",
```

Comment:
Why `2.1.16`? Latest AET release is `2.1.6` and currently we have `2.1.7-SNAPSHOT` on master branch